src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -112,36 +156,31 @@ private static bool Quit(string title, string text)
             return n == 0;
         }
 
-        private static string GetPaddedString(List<string> strings, int maxWidth, int offset = 0)
+        private static string GetPaddedString(List<string> strings, int[] colWidths, int offset = 0)
         {
-            int colWidth = maxWidth / strings.Count;
             var builder = new StringBuilder();
-            foreach (var str in strings)
+            if (offset > 0)
             {
-                // If the string won't fit in the column, append an ellipsis.
-                if (str.Length >= colWidth)
+                builder.Append(string.Empty.PadRight(offset));
+            }
+
+            for (int i = 0; i < strings.Count; i++)
+            {
+                if (i > 0)
```

Comment:
This is to add a single space between each column, right? Can you leave a small comment saying so?

Reply:
Yes